_pkgdown.yml

```diff
@@ -0,0 +1,7 @@
+home:
+  title: Tools for working with geohashes
+authors:
+  Michael Chirico:
+    href: https://github.com/MichaelChirico
+template:
+  params:
```

Comment:
for the `pkgdown` site, I would deploy it on GitHub pages:

https://pkgdown.r-lib.org/reference/deploy_site_github.html

Reply:
Fixed in [a3c4fde](https://github.com/MichaelChirico/geohashTools/pull/33/commits/a3c4fde3d47ee389fe3e4b5fc95fff7fe6747995)